packages/extra/src/Temporals.js

```diff
@@ -0,0 +1,296 @@
+/**
+ * @copyright (c) 2016, Philipp Thürwächter & Pattrick Hüper & Michał Sobkiewicz
```

Comment:
In this case, since you are the main contributor to the extra package and creator of this file, i think its fine to add your name, but what about to change the date to `2015-present` ?

Reply:
Sure, I'll change the date.